pollbot/views/release.py

```
@@ -22,7 +22,7 @@ def status_response(task):
 
         try:
             response = await task(product, version)
-        except TaskError as e:
+        except (TaskError, ClientError) as e:
```

Are there exception types that we *shouldn't* render this kind of JSON exception for? What do we normally do if we get an exception? The issue doesn't really explain why you would want this..

---

I believe we mainly want timeout or 5XX errors on the remote service to be reported cleanly

---

That's a really good reason, should we report an errors status for all kinds of task errors?

I guess it is ok to return a 503 if the task is failing because the remote is broken.
However it is also good to return a task error because PollBot didn't fail, that's just the service it wanted to use to check the task.

I like the idea of handling all kinds of Exception here.